java/compaction-job-execution/src/test/java/sleeper/compaction/jobexecution/CompactSortedFilesRunnerIT.java

```diff
@@ -155,6 +156,7 @@ public void shouldDeleteMessages() throws IOException, StateStoreException, Obje
         stateStore.initialise();
         DynamoDBCompactionJobStatusStoreCreator.create(instanceProperties, dynamoDB);
         CompactionJobStatusStore jobStatusStore = DynamoDBCompactionJobStatusStore.from(dynamoDB, instanceProperties);
+        CompactionTaskFinishedStatus.Builder taskFinishedBuilder = CompactionTaskFinishedStatus.builder();
```

Comment:
I'd find it a bit clearer if this was inlined into the constructor call, so you don't need to check if this variable is used anywhere.